employee_dialog/lib/main.dart

```diff
@@ -1,94 +1,31 @@
+import 'package:employee/Employee profile/employee_preferences.dart';
 import 'package:flutter/material.dart';
-import 'employee.dart';
-import 'employee_detail.dart';
+import 'package:flutter/services.dart';
+import 'Employee profile/profile_page.dart';
 
-void main(){
-  runApp(const EmployeeApp());
-}
+Future main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  await SystemChrome.setPreferredOrientations([
```

Comment:
Do we need to set device orientations? If anything, we should build the application in a manner that we allow the usage of both portrait and landscape mode.